ui/src/alerting/components/AlertColumnHeader.tsx

```diff
@@ -0,0 +1,27 @@
+// Libraries
+import React, {FC} from 'react'
+
+// Components
+import {Button, IconFont} from '@influxdata/clockface'
+
+const style = {
+  display: 'flex',
+  justifyContent: 'space-between',
+  alignItems: 'center',
+}
+
+interface Props {
+  title: string
+  onCreate: () => void
+}
+
+const AlertsColumnHeader: FC<Props> = ({onCreate, title}) => {
```

Comment:
the file name and component name don't match (one's alerts.. the other is alert.. )and that can get confusing, could they match please? :)

Reply:
yaaas!

Reply:
good catch